canister/src/api/fee_percentiles.rs

```diff
@@ -419,8 +419,11 @@ mod test {
     fn get_current_fee_percentiles_from_utxos() {
         let number_of_blocks = 5;
         let number_of_transactions = 10_000;
-        let blocks = generate_blocks(10_000, number_of_blocks);
-        let stability_threshold = 1;
+        let blocks = generate_blocks(10_000, number_of_blocks)
```

Comment:
Do we need this change (and similarly below)?

Reply:
I think so, I remember those tests were failing without it.